playbooks/inventory.yml

```diff
@@ -1,10 +1,7 @@
 ---
 all:
   hosts:
-    dev:
-      ansible_host: 127.0.0.1
-      ansible_connection: local
-  children:
-    vagrant:
-      hosts:
-        dev:
+    acid-server:
+      ansible_host:
```

Comment:
Leave a hint of what should be here.
For example:
```yaml
acid-server:
  # Where to deploy ACID dashboard
  ansible_host: 10.100.0.1
```
Don't hesitate to use `10.100.0.x` addresses as example.